.github/actions/cache/src/utils.js

```diff
@@ -0,0 +1,63 @@
+const core = require('@actions/core')
+const fs = require('fs')
+const path = require('path')
+
+async function getSortedCacheFiles(cachePath, key = '') {
+  if (!fs.existsSync(cachePath)) {
```

Comment:
In case of usage into async function, make sense to use [fs.stat()](https://nodejs.org/docs/latest/api/fs.html#fsstatpath-options-callback) or [fs.access()](https://nodejs.org/docs/latest/api/fs.html#fsaccesspath-mode-callback) instead. 
`fs.exists` - is deprecated.

Reply:
replaced with access and stat